GopherCAN.c

```diff
@@ -637,12 +638,17 @@ void service_can_tx_hardware(CAN_HandleTypeDef* hcan)
 #endif
 
 	// add messages to the the TX mailboxes until they are full
-	while (!is_empty(buffer) && HAL_CAN_GetTxMailboxesFreeLevel(hcan))
+	while (!IS_EMPTY(buffer) && HAL_CAN_GetTxMailboxesFreeLevel(hcan))
 	{
 		U32 tx_mailbox_num;
 
+		// Turn off the TX interrupt (if applicable)
+#if TARGET == F7XX || TARGET == F4XX
+		HAL_CAN_DeactivateNotification(hcan, CAN_IT_TX_MAILBOX_EMPTY);
+#endif
```

Comment:
Nit: I think it's good style to inline comment what the `#endif` is for, like so
```
#if TARGET == F7XX || TARGET == F4XX
		HAL_CAN_DeactivateNotification(hcan, CAN_IT_TX_MAILBOX_EMPTY);
#endif // TARGET == F7XX || TARGET == F4XX
```
again, just a style thing, but it makes it a lot easier in dense `#define` blocks